sympy/stats/crv_types.py

```diff
@@ -1379,8 +1395,23 @@ def pdf(self, x):
         k, theta = self.k, self.theta
         return x**(k - 1) * exp(-x/theta) / (gamma(k)*theta**k)
 
-    def sample(self):
-        return random.gammavariate(self.k, self.theta)
+    def _sample_random(self, size):
```

Comment:
I would call this `_sample_python`, to make it clear that it uses the Python `random` standard library.